feature/danggn/src/main/java/com/mashup/feature/danggn/ranking/DanggnWeeklyRankingContent.kt

```diff
@@ -0,0 +1,136 @@
+package com.mashup.feature.danggn.ranking
+
+import android.graphics.Paint.Cap
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.mashup.core.common.R
+import com.mashup.core.ui.colors.Brand500
+import com.mashup.core.ui.colors.Gray100
+import com.mashup.core.ui.colors.RankingGrayColor
+import com.mashup.core.ui.theme.MashUpTheme
+import com.mashup.core.ui.typography.Caption3
+import com.mashup.core.ui.typography.Title1
+import com.mashup.core.ui.typography.Title3
+
+@Composable
+fun DanggnWeeklyRankingContent(
+    modifier: Modifier = Modifier,
+
+    ) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Color.White),
```

Comment:
```suggestion
            .background(Gray50),
```

Gray50을 사용해야 할 거 같아요!

Reply:
굿굿 감사링